i18n/es.po

```diff
@@ -497,7 +497,7 @@ msgid "Invalid output combo"
 msgstr ""
 
 msgid "Scheduler"
-msgstr ""
+msgstr "Programador"
```

Comment:
Question for @MiquelAdell: Maybe we can use "Planificador" here since it's similar to the translation used by DHIS in the menu.

![image](https://github.com/EyeSeeTea/predictor-extended-app-dev/assets/461124/857f5a04-7ff7-4068-a55c-836e79ef1a45)